doc/releases/changelog-dev.md

```diff
@@ -187,27 +187,39 @@
   }
   ```
 
-  * A new :func:`~catalyst.passes.decompose_arbitrary_ppr` pass has been added to the `catalyst.passes` module.
-    This will decompose an arbitrary angle PPR into a collection of PPRs, PPMs and
-    a single-qubit arbitrary PPR in the Z basis.
-    [(#2304)](https://github.com/PennyLaneAI/catalyst/pull/2304)
-
-    ```python
-      import pennylane as qml
-      from catalyst import qjit, measure
-      from catalyst.passes import decompose_arbitrary_ppr, to_ppr
-
-      @qjit(pipelines=[("pipe", ["quantum-compilation-stage"])], target="mlir")
-      @decompose_arbitrary_ppr
-      @to_ppr
-      @qml.qnode(qml.device("null.qubit", wires=3))
-      def circuit():
-          qml.PauliRot(0.123, pauli_word="XXY", wires=[0, 1, 2])
-          return
-
-      print(circuit.mlir_opt)
-    ```
+* A new transform called :func:`~catalyst.passes.decompose_arbitrary_ppr` pass has been added, 
+  which decomposes abitrary-angle Pauli product rotations (PPRs) as outlined in Figure 13(d) from 
+  `arXiv:2211.15465 <https://arxiv.org/abs/2211.15465>`_.
+  [(#2304)](https://github.com/PennyLaneAI/catalyst/pull/2304)
 
+  The ``decompose_arbitrary_ppr`` will decompose an arbitrary angle PPR into a collection of PPRs, 
+  PPMs and a single-qubit arbitrary PPR in the ``Z`` basis:
```

Comment:
I know saying arbitrary-angle PPR is quite cumbersome, but I think arbitrary PPR by itself doesn't really mean much? ^^

Reply:
I've been thinking about alternatives, like continuous/discrete PPRs, but I don't know.

Reply:
I know 😭 I mulled over several names but they were all too long and I eventually came back to Sengthai's original name hahaha